Gemfile

```diff
@@ -91,10 +91,10 @@ end
 
 # Bulkrax
 group :bulkrax do
-  # Until d36cf3606545ea30da8d8082f1b67b96d9aaf8c1 is in a major release, use this ref or a ref
+  # Until d1d0eca5963fcd190955ab75bcf9d0285afa2bb2 is in a major release, use this ref or a ref
   # who's ancestors include this ref.
   # rubocop:disable Metrics/LineLength
-  gem 'bulkrax', "~> 4.4", git: "https://github.com/samvera-labs/bulkrax.git", ref: "d36cf3606545ea30da8d8082f1b67b96d9aaf8c1"
+  gem 'bulkrax', "~> 4.4", git: "https://github.com/samvera-labs/bulkrax.git", ref: "d1d0eca5963fcd190955ab75bcf9d0285afa2bb2"
```

Comment:
I don't think ive seen ref's like this before. Does this point to a bulkrax commit or something?

Reply:
This does point to a specific commit on the main branch.

Reply:
Oh cool! Ty :)